app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -165,24 +156,6 @@ protected void redirectIfNotLoggedIn() {
         }
     }
 
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
-        TypedValue value = new TypedValue();
-        int color;
-        if (getTheme().resolveAttribute(R.attr.toolbar_icon_tint, value, true)) {
-            color = value.data;
-        } else {
-            color = Color.WHITE;
-        }
-        for (int i = 0; i < menu.size(); i++) {
-            Drawable icon = menu.getItem(i).getIcon();
-            if (icon != null) {
-                icon.setColorFilter(color, PorterDuff.Mode.SRC_ATOP);
-            }
-        }
-        return super.onCreateOptionsMenu(menu);
-    }
-
```

Comment:
this was once needed, but with the new Material themes it apparently is no longer